src/components/BusInfoCarousel.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Slider from 'react-slick';
+import BusInfoContainer from './BusInfoContainer';
+import { IoIosArrowForward, IoIosArrowBack } from 'react-icons/io';
+
+export default class BusInfoCarousel extends Component {
```

Comment:
maybe its better if this was functional component instead of class

Reply:
this is a good point, but sometimes this happens when refactoring is more time-consuming. This component will be edited later